https://issues.redhat.com/browse/ACM-16367--MCE install cannot be on … (#7420)
open-cluster-management/rhacm-docs: clusters/install_upgrade/install_connected.adoc

*Important:* 

> - You cannot install {mce-short} on a cluster that has a `ManagedCluster` resource. You must remove the `ManagedCluster` if it exists on the cluster before you can install {mce-short}.

Reviewer: I think this is a bit ambiguous. I think we could make it clearer with something like the following:
```suggestion
- You cannot install {mce-short} on a cluster that has a `ManagedCluster` resource configured in an external cluster. You must remove the `ManagedCluster` from the external cluster before you can install {mce-short}.
```
if the words "hub cluster" are problematic, you can use external cluster but hub cluster is also valid

Author: hub cluster is fine bc it is an OCP hub cluster.

> - You cannot install {mce-short} on a cluster that has a `ManagedCluster` resource configured in an external cluster. You must remove the `ManagedCluster` from the external cluster before you can install {mce-short}.

Reviewer 2: ```suggestion
- You cannot install {mce-short} on a cluster that has a `ManagedCluster` resource configured in an external cluster. You must remove the `ManagedCluster` resource from the external cluster before you can install {mce-short}.
```

[#connect-prerequisites-mce]
== Prerequisites

Before you install {mce-short}, see the following prerequisites:

* Your {ocp-short} cluster must have access to the {mce-short} in the OperatorHub catalog from the console. 

* You need access to the link:https://catalog.redhat.com/software/containers/search?p=1&application_categories_list=Container%20Platform%20%2F%20Management[catalog.redhat.com].

> * Your cluster is _not_ managed as a `local-cluster`.

Reviewer: ```suggestion
* Your cluster does _not_ have a ManagedCluster configured in an external cluster.
```

> * Your cluster does _not_ have a `ManagedCluster` configured in an external cluster.

Reviewer 2: ```suggestion
* Your cluster does _not_ have a `ManagedCluster` resource configured in an external cluster.
```
